From a9e98df117fe9f8901783eaa5b408ab1edb4c5b1 Mon Sep 17 00:00:00 2001
From: Riccardo Schirone <sirmy15@gmail.com>
Date: Mon, 26 Nov 2018 13:57:33 +0100
Subject: [PATCH] io_ptrace: do not include io_ptrace if DEBUGGER is not
 defined

---
 libr/io/p/io_ptrace.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libr/io/p/io_ptrace.c b/libr/io/p/io_ptrace.c
index 245687cf9..435d5d5df 100644
--- a/libr/io/p/io_ptrace.c
+++ b/libr/io/p/io_ptrace.c
@@ -7,7 +7,7 @@
 #include <r_cons.h>
 #include <r_debug.h>
 
-#if __linux__ || __BSD__
+#if DEBUGGER && (__linux__ || __BSD__)
 
 #include <sys/ptrace.h>
 #include <sys/types.h>
-- 
2.19.1

