From 78792d4c8eb8fc7891563ffafbff17da484b7360 Mon Sep 17 00:00:00 2001
From: Riccardo Schirone <sirmy15@gmail.com>
Date: Thu, 6 Sep 2018 09:31:48 +0200
Subject: [PATCH] flag/d: explicitly name files, to ensure reproducibility

---
 libr/flag/d/meson.build | 12 +++++++++---
 1 file changed, 9 insertions(+), 3 deletions(-)

diff --git a/libr/flag/d/meson.build b/libr/flag/d/meson.build
index ac94cc4c2..67df05f64 100644
--- a/libr/flag/d/meson.build
+++ b/libr/flag/d/meson.build
@@ -1,11 +1,17 @@
-files = run_command(glob_cmd + ['*']).stdout().strip().split(';')
+files = [
+  'alloc',
+  'env',
+  'fs',
+  'network',
+  'process',
+  'stdout',
+  'string'
+]
 
 script = '''#script
 import os
 from sys import argv
 for fname in argv[1:]:
-    if any(ex in fname for ex in ('meson.build', 'Makefile')):
-        continue
     with open(fname) as f:
         text = ' '.join(f.read().splitlines())
     print('ft %s %s' % (os.path.basename(fname), text))
-- 
2.14.4

