From f81e5622d163b173ca2bb3e8998debf9be0fb0c0 Mon Sep 17 00:00:00 2001
From: Riccardo Schirone <sirmy15@gmail.com>
Date: Tue, 16 Oct 2018 10:31:02 +0200
Subject: [PATCH] r_util: use char * instead of void *

---
 libr/include/r_util/r_sys.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libr/include/r_util/r_sys.h b/libr/include/r_util/r_sys.h
index 924ce3015..861604e04 100644
--- a/libr/include/r_util/r_sys.h
+++ b/libr/include/r_util/r_sys.h
@@ -98,7 +98,7 @@ R_API bool r_sys_tts(const char *txt, bool bg);
 #define r_sys_breakpoint() __asm__ volatile ("bkpt $0");
 #else
 #warning r_sys_breakpoint not implemented for this platform
-#define r_sys_breakpoint() { void *a = NULL; *a = 0; }
+#define r_sys_breakpoint() { char *a = NULL; *a = 0; }
 #endif
 #endif
 
-- 
2.14.4

