diff -up radare2-3.2.0/meson.build.orig radare2-3.2.0/meson.build
--- radare2-3.2.0/meson.build.orig	2019-01-10 09:38:46.353532035 +0100
+++ radare2-3.2.0/meson.build	2019-01-10 09:39:18.732366400 +0100
@@ -59,6 +59,11 @@ endif
 if get_option('r2_gittap') != ''
   gittap = get_option('r2_gittap')
 endif
+# gittap is used for the version of each r_ library
+# in case it has not been set (e.g. a release tarball) set it
+if gittap == ''
+  gittap = r2_version
+endif
 
 if get_option('r2_gittip') != ''
   gittip = get_option('r2_gittip')
